src/cpp/src/icontinuous_batching.cpp

```diff
@@ -76,7 +78,16 @@ ContinuousBatchingPipeline::IContinuousBatchingPipeline::generate(
             res.m_status
         });
     }
+    auto stop_time = std::chrono::steady_clock::now();
+
+    auto& raw_counters = perf_metrics.raw_metrics;
+    raw_counters.generate_durations = std::vector<MicroSeconds>();
+    raw_counters.generate_durations.emplace_back(PerfMetrics::get_microsec(stop_time - start_time));
+    raw_counters.tokenization_durations.emplace_back(PerfMetrics::get_microsec(decode_start_time - start_time));
```

Comment:
looks like this value is incorrect as it counts `generate` time as well

you need to introduce `encode_stop_time`

Reply:
Agreed. Implemented correct tok/detokenize calcualation in the lates commit